tests/e2e/tests/regression/browser/database-list-search.e2e.ts

```diff
@@ -0,0 +1,104 @@
+import { t } from 'testcafe';
+import { acceptLicenseTerms } from '../../../helpers/database';
+import { addNewStandaloneDatabase, addNewOSSClusterDatabase, deleteDatabaseByName } from '../../../helpers/api/api-database';
+import { MyRedisDatabasePage } from '../../../pageObjects';
+import { rte } from '../../../helpers/constants';
+import { commonUrl, ossStandaloneConfig, ossStandaloneV5Config, ossClusterConfig } from '../../../helpers/conf';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const databasesForSearch = [
+    { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port, databaseName: 'testSearch' },
+    { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port, databaseName: 'testSecondSearch' },
+    { host: ossStandaloneV5Config.host, port: ossStandaloneV5Config.port, databaseName: 'testV5' },
+    { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port, databaseName: 'lastConnection' }
+]
+
+fixture`Database list search`
+    .meta({ type: 'regression' })
+    .page(commonUrl)
+    .beforeEach(async () => {
+        // Add new databases using API
+        await acceptLicenseTerms();
+        await addNewStandaloneDatabase(ossStandaloneConfig);
+        await addNewStandaloneDatabase(databasesForSearch[0]);
+        await addNewStandaloneDatabase(databasesForSearch[1]);
+        await addNewStandaloneDatabase(databasesForSearch[2]);
```

Comment:
You could use a loop, or, consider creating a function of `addNewStandaloneDatabases`

Reply:
fixed, function addNewStandaloneDatabases added